LICENSES/MIT.txt

```diff
@@ -0,0 +1,19 @@
+MIT License Copyright (c) <year> <copyright holders>
```

Comment:
I have no clue what you could do with that line but leaving the default in seems strange.

Reply:
As I, not a lawyer, understand it: the files in the `LICENSES/` directory are just for reference, because you MUST provide the license with the project. They shouldn't be changed, because quick check with `sha256sum` allows to verify if the license was changed or not

 Also, if I replaced the example text here with say, current year and my name and email, that probably would mean that suddenly DebOps is licensed under MIT license, right? Without changing the details here, it probably means that the MIT license applies only to the files that explicitly refer to it.

Reply:
I think the text you linked is about the legal text in the license which talks about your rights not the header. So lines 3 to 19 should stay the same. I think the best choice would be to drop the year and holders completely from this file.

Also sha sums are irrelevant cause as soon as you change any whitespace it is different which does not change the license what so ever.

Reply:
I removed the `Copyright (c) <year> <copyright holders>` string from the file and the `reuse` script doesn't scream about any issues, so I suppose that's fine.